plasmapy/formulary/lengths.py

```diff
@@ -175,13 +184,13 @@ def gyroradius(
     radius and is given by
 
     .. math::
-        r_{Li} = \frac{V_{\perp}}{ω_{ci}}
+        r_{Li} = \frac{\gamma V_{\perp}}{ω_{ci}}
 
     where :math:`V_⟂` is the component of particle velocity that is
-    perpendicular to the magnetic field and :math:`ω_{ci}` is the
-    particle gyrofrequency.  If a temperature is provided, then
+    perpendicular to the magnetic field, :math:`ω_{ci}` is the
+    particle gyrofrequency, and :math: `γ` is the lorentz factor.  If a temperature is provided, then
     :math:`V_⟂` will be the most probable thermal velocity of a
-    particle at that temperature.
+    particle at that temperature. :math: `γ` can manually be set to 1 to avoid the relativistic correction
```

Comment:
```suggestion
    particle at that temperature. :math: `γ` can manually be set
    to 1 to avoid the relativistic correction.
```
...since there's a PEP 8 style guideline to keep docstrings and comments at ≤ 72 characters per line.